bindings/core/src/method_handler/client.rs

```diff
@@ -80,7 +80,7 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
                 output_builder = output_builder.with_immutable_features(immutable_features)
             }
 
-            Response::Output(output_builder.finish_output()?)
```

Comment:
Why not add `Output(#[from] OutputError),`to the error Enum so we don't have to do .map_err()?

Reply:
could do that I suppose